HW1/P2/P2b.py

```diff
@@ -1,3 +1,21 @@
 from P2 import *
+import findspark
+findspark.init()
+import os
+import pyspark
+sc = pyspark.SparkContext()
 
-# Your code here
+# define shuffled axis: to ensure randomness, the axes are defined separately
+axis1 = range(2000)
+np.random.shuffle(axis1)
+axis2 = range(2000)
+np.random.shuffle(axis2)
```

Comment:
By shuffling the data, you achieved your goal of randomizing the partitioning. But this was enabled because you knew how the data was structured. You won't always have that luxury. There are other methods to randomize partitioning that are more general (like .partitionBy() where you pass in a hash function or .repartition()) and more "spark-like".